### PR TITLE
FIX: vertices

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR fixes hemi logic in stc.

Closes https://github.com/mne-tools/mne-python/issues/8609

### CODE REVIEW

**Comment by larsoner** [comment]
Can you add a unit test that would have caught this? Having to hear about these sorts of problems from CircleCI is not as nice as local `pytest mne/viz`

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -1907,8 +1907,8 @@ def _plot_stc(stc, subject, surface, hemi, colormap, time_label,
     for hi, hemi in enumerate(hemis):
```
Comment (drammock) [inline mne/viz/_3d.py:1907]: I think with this change you no longer need `hi` or `enumerate`

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
@GuillaumeFavelier We got a segfault on Py3.8, not sure if related?

**Comment by GuillaumeFavelier** [comment]
I'll investigate because it's close to the test I modified.

**Comment by larsoner** [comment]
Restarted -- I think I've seen a spurious macOS test failure like this before. Probably something having to do with closing windows / garbage collection / Qt / VTK

**Review by agramfort — APPROVED** [review]

**Comment by larsoner** [comment]
Thanks @GuillaumeFavelier ! Hopefully CircleCI goes back to green now

**Pull request merged**